sechub-developertools/scripts/sechub-api.sh

```diff
@@ -268,6 +293,57 @@ function sechub_alive_check {
 }
 
 
+function sechub_asset_file_create {
+  local asset_id="$1"
+  local asset_file="$2"
+  local asset_file_sha256=`sha256sum "$asset_file" | cut -d ' ' -f 1`
```

Comment:
```suggestion
  local asset_file_sha256=$(sha256sum "$asset_file" | cut -d ' ' -f 1)
```

Reply:
The two forms are equivalent.
Leaving them as they are.